src/content/self-hosted/helm-configuration.mdx

```diff
@@ -754,14 +770,24 @@ globals:
 
 #### priorityClassName
 
-Defines the priority class to be used by all pods in the okteto namespace.
+Defines the priority class to be used by all pods in the Okteto Namespace.
```

Comment:
```suggestion
Defines the priority class to be used by all pods in the `okteto` namespace. 
```
We use upper case for namespaces managed by Okteto, but we are referring to a Kubernetes namespace here